.github/workflows/e2e.yml

```
@@ -147,6 +147,18 @@ jobs:
       DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
       DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
 
+  e2e-vcluster:
+    if: ${{ inputs.e2e_test_suites == 'all' || inputs.e2e_test_suites == 'hostpath' || inputs.e2e_test_suites == '' }}
+    needs: [build] 
+    uses: ./.github/workflows/e2e-vcluster.yml
+    with:
+      vlogger-image: ${{ needs.build.outputs.vlogger-image }}
+      operator-image: ${{ needs.build.outputs.operator-image }}
+      vertica-image: ${{ needs.build.outputs.full-vertica-image }}
```

We need to pass in the v2 image here.
```suggestion
      vertica-image: ${{ needs.build.outputs.v2-vertica-image }}
```